docs/.vitepress/versions.ts

```diff
@@ -1,7 +1,25 @@
+import { execSync } from 'child_process';
 import { version } from '../../package.json';
 
-export const currentVersion = `v${version}`;
+const NEXT_TEXT = 'Next';
+
+function readBranchName() {
+  try {
+    return execSync('git branch --show-current').toString('utf8') || NEXT_TEXT;
+  } catch {
+    return NEXT_TEXT;
+  }
+}
+
+const branchName = readBranchName();
+const isNext = !branchName.match(/^v\d+$/);
+
+export const currentVersion = isNext ? NEXT_TEXT : `v${version}`;
+
+const nextVersion = { version: NEXT_TEXT, link: 'https://next.fakerjs.dev/' };
+const latestVersion = { version: `v${version}`, link: 'https://fakerjs.dev/' };
 
 export const oldVersions = [
+  isNext ? latestVersion : nextVersion,
   { version: 'v6.3.1', link: 'https://v6.fakerjs.dev/' },
```

Comment:
Do we already also need to add v7 as we are in v8 now? (I mean in a separate PR)

```suggestion
  isNext ? latestVersion : nextVersion,
  { version: 'v7.6.0', link: 'https://v7.fakerjs.dev/' },
  { version: 'v6.3.1', link: 'https://v6.fakerjs.dev/' },
```

I think this is needed at the time when we switch branches in that way that v7.fakerjs.dev already exists :thinking:

Reply:
This is not possible, because then once we release v8 it will point to v8, 
but the main branch cannot be used with netlify/and it would still point to the same version.
The test ensures that we don't forget to add v7 once v8 is released.

As for automating this, IMO this is not worth it since it only changes very rarely (once every few months).

Reply:
Partly I see what you mean
But I see an issue if we want to fully automate the release process :thinking: But might be solved by just using the right workflow
We might need to discuss about that in person

Reply:
The release process could add the required changes, since it also decides the version and the tag that the docs will reference.

Reply:
release-it supports hooks, so yeah we can add scripts how ever we want to :thinking:

Reply:
After a nights worth of thoughts, I think I found an easy way to automate this in every docs build.
Let me check first, whether my idea works in reality as well.

Reply:
Done